### PR TITLE
fix: fix hot reload serving of static resources

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Fixes an issue where we weren't actually using an overridden dev server path in our local development workflows.

JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)

**What changed?**

- Edited loading of static scripts to be aware of base URL overrides.
- Edited logic to correctly handle if loaded script is a root URL or a relative URL.

**Developer Checklist**

- [ ] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

- Follow developers guide to get a local install of ORA running, up to and including the *Hot Reload Frontend Changes* section.
- Verify that changes to JS are getting loaded and live reloaded correctly.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Review by leangseu-edx — APPROVED** [review]
